src/components/Dropdown.tsx

```diff
@@ -0,0 +1,103 @@
+import React, { useCallback, useState } from 'react';
+import debounce from 'lodash/debounce';
```

Comment:
Still not fixed. commit your package.json flie
I'd recomment to write own debounce function